variables.tf

```diff
@@ -3,11 +3,13 @@
 
 # Provider parameters
 variable "api_fingerprint" {
+  default     = ""
   description = "Fingerprint of oci api private key."
   type        = string
 }
 
 variable "api_private_key_path" {
```

Comment:
can we add one more variable for private_key in addition to private key path that will be useful in CI/CD.

Reply:
Also lets use the same name as in provider instead of api_fingerprint etc..

Reply:
ok let's do that separately